README.md

```diff
@@ -9,3 +9,22 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+Direct Proof:
+- If we have two graphs $H_{1}$ and $H_{2}$.
+  - $H_{1}$ has 3 vertices $(A, B, C)$ and one edge $(A, B)$.
+  - $H_{2}$ has 4 vertices $(1, 2, 3, 4)$ and one edge $(1, 2)$.
+- There is no one-to-one and onto mapping between $H_{1}$ and $H_{2}$.
+- There is no vertex in $H_{1}$ that can correspond to 4 in $H_{2}$.
+- This violates the bijection requirement.
+  - There are more vertices in $H_{2}$ than in $H_{1}$. This makes it impossible for every vertex in $H_{2}$ to be mapped to by a vertex in $H_{1}$.
+  - This breaks the onto condition of bijection.
+-  This shows that two graphs that do not have the same number of nodes cannot be isomorphic.
```

Comment:
What about any other two graphs that don't have the same number of nodes?

Reply:
Or do you want me to also mention about the one to one condition being violated it the graphs have a different number of nodes but are still onto?

Reply:
At the moment you don't have a general proof because you're using a specific example. It needs to be a general proof.

Reply:
I believe I made the appropriate adjustments.